Block/Adminhtml/Plan/Edit/Form.php

```diff
@@ -0,0 +1,168 @@
+use \Magento\Backend\Block\Widget\Form\Generic;
+ 
+class Form extends Generic
+{
+ 
+    /**
+     * @var \Magento\Store\Model\System\Store
+     */
+    protected $_systemStore;
+    /**
+     * @var \Magento\Store\Model\System\Store
+     */
+    protected $_status;
+    /**
+     * @var \Magento\Store\Model\System\Store
+     */
+    protected $_productList;
+ 
+    /**
+     * @param \Magento\Backend\Block\Template\Context $context
+     * @param \Magento\Framework\Registry $registry
+     * @param \Magento\Framework\Data\FormFactory $formFactory
+     * @param \Magento\Store\Model\System\Store $systemStore
+     * @param \Razorpay\Subscription\Model\Source\Plan\Status $status
+     * @param \Razorpay\Subscription\Model\Source\ProductList $productList
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Backend\Block\Template\Context $context,
+        \Magento\Framework\Registry $registry,
+        \Magento\Framework\Data\FormFactory $formFactory,
+        \Magento\Store\Model\System\Store $systemStore,
+        \Razorpay\Subscription\Model\Source\Plan\Status $status,
+        \Razorpay\Subscription\Model\Source\ProductList $productList,
+        array $data = []
+    ) {
+        $this->_systemStore = $systemStore;
+        $this->_status = $status;
+        $this->_productList = $productList;
+        parent::__construct($context, $registry, $formFactory, $data);
+    }
+ 
+    /**
+     * Init form
+     *
+     * @return void
+     */
+    protected function _construct()
+    {
+        parent::_construct();
+        $this->setId('plan_form');
+        $this->setTitle(__('Plan Information'));
+    }
+ 
+    /**
+     * Prepare form
+     *
+     * @return $this
+     */
+    protected function _prepareForm()
+    {
+        /** @var \Razorpay\Subscription\Model\Plans $model */
+        $model = $this->_coreRegistry->registry('subscribed_plan');
+ 
+        /** @var \Magento\Framework\Data\Form $form */
+        $form = $this->_formFactory->create(
+            ['data' => ['id' => 'edit_form', 'action' => $this->getData('action'), 'method' => 'post']]
+        );
+ 
+        $form->setHtmlIdPrefix('plan_');
+ 
+        $fieldset = $form->addFieldset(
+            'base_fieldset',
+            ['legend' => __('Plan Details'), 'class' => 'fieldset-wide']
+        );
+ 
+        if ($model->getId()) {
+            $fieldset->addField('entity_id', 'hidden', ['name' => 'entity_id']);
+        }
+ 
+        $fieldset->addField(
+            'plan_name',
+            'text',
+            ['name' => 'plan_name', 'label' => __('Plan Name'), 'title' => __('Plan Name'), 'required' => true]
+        );
+ 
+        $fieldset->addField(
+            'plan_desc',
+            'textarea',
+            ['name' => 'plan_desc', 'label' => __('Plan Description'), 'title' => __('Plan Description'), 'required' => true]
+        );
+        // $fieldset->addField(
+        //     'magento_product_id',
+        //     'text',
```

Comment:
Remove this commented code from every where if not required

Reply:
done